universal-application-tool-0.0.1/app/controllers/admin/AdminApplicationController.java

```diff
@@ -64,7 +64,8 @@ public AdminApplicationController(
   @Secure(authorizers = Authorizers.Labels.ANY_ADMIN)
   public Result downloadAll(Http.Request request, long programId) {
     try {
-      ProgramDefinition program = programService.getProgramDefinition(programId);
+      ProgramDefinition program =
+          programService.getProgramDefinition(programId, /* version = */ null);
```

Comment:
Probably better to do Optional.empty rather than null.

Reply:
Good call. I removed the version altogether.